src/accessibility/sonification/index.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+var c2m = require('chart2music');
+var Fx = require('../../components/fx');
+var Lib = require('../../lib');
+
+var codecs = require('./all_codecs').codecs;
+
+function initC2M(gd, defaultConfig) {
+    // TODO: should this be Lib.getGraphDiv()?
```

Comment:
@archmoj or @alexcjohnson might be better equipped to answer this?

Reply:
Passing `gd` as an argument to this function (as implemented) is better that getting it from its id using `Lib.getGraphDiv`.